brainwidemap/encoding/pipelines/03_gather_results.py

```diff
@@ -0,0 +1,140 @@
+# Standard library
+from functools import cache
+
+# Third party libraries
+import numpy as np
+import pandas as pd
+
+# IBL libraries
+from ibllib.atlas import BrainRegions
+from iblutil.numerical import ismember
+
+# Brainwidemap repo imports
+from brainwidemap.encoding.utils import get_id, remap
+
+
+def colrename(cname, suffix):
+    return str(cname + 1) + "cov" + suffix
+
+
+def remap(ids, source="Allen", dest="Beryl", output="acronym", br=BrainRegions()):
+    _, inds = ismember(ids, br.id[br.mappings[source]])
+    ids = br.id[br.mappings[dest][inds]]
+    if output == "id":
+        return br.id[br.mappings[dest][inds]]
+    elif output == "acronym":
+        return br.get(br.id[br.mappings[dest][inds]])["acronym"]
+
+
+def get_id(acronym, brainregions=BrainRegions()):
+    return brainregions.id[np.argwhere(brainregions.acronym == acronym)[0, 0]]
+
+
+def get_name(acronym, brainregions=BrainRegions()):
+    if acronym == "void":
+        return acronym
+    reg_idxs = np.argwhere(brainregions.acronym == acronym).flat
+    return brainregions.name[reg_idxs[0]]
+
+
+def label_cerebellum(acronym, brainregions=BrainRegions()):
+    regid = brainregions.id[np.argwhere(brainregions.acronym == acronym).flat][0]
+    ancestors = brainregions.ancestors(regid)
+    if "Cerebellum" in ancestors.name or "Medulla" in ancestors.name:
+        return True
+    else:
+        return False
+
+
+if __name__ == "__main__":
+    # Standard library
+    import os
+    import pickle
+    from pathlib import Path
+
+    # Brainwidemap repo imports
+    # Brainwide repo imports
+    from brainwidemap.encoding.params import GLM_CACHE, GLM_FIT_PATH
+
+    currdate = "2023-01-16"  # Date on which fit was run
```

Comment:
Would be good to move into params as well, or somehow make sure it is shared between these three scripts (I assume it is always the same?)

Reply:
It's not always the same, and changes run-to-run. I use the date, coupled with an archival system of annotated run files, to store past sets of parameters. However it does make more sense to have this as an argument using argparse, which I'm switching to.